.github/workflows/snyk-python-scan.yml

```
@@ -0,0 +1,61 @@
+name: 🚨 Snyk Python Scan
+
+ on:
+   push:
+     branches:
+     - long_lived/**
+     - main
+     - release/**
+     tags:
+     - '**'
+   pull_request:
+     branches:
+     - '**'
```

Remove extra leading space.

```suggestion
on:
  push:
    branches:
    - long_lived/**
    - main
    - release/**
    tags:
    - '**'
  pull_request:
    branches:
    - '**'
  workflow_dispatch: null
```